src/Items/ItemSeeds.h

```diff
@@ -37,7 +37,7 @@ class cItemSeedsHandler :
 	{
 		switch (m_ItemType)
 		{
-			case E_ITEM_CARROT: return FoodInfo(3, 4.8);
+			case E_ITEM_CARROT: return FoodInfo(3, 3.6);
```

Comment:
The food info is full of these magic numbers, they represent the hunger bar replenished by eating this item; I don't think it's necessary to mask these beyond another layer of names. Keep this as-is.